auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java

```diff
@@ -280,6 +281,10 @@ public boolean isAWSDiskEncryptionWithCMKEnabled(String accountId) {
         return isEntitlementRegistered(accountId, CDP_CB_AWS_DISK_ENCRYPTION_WITH_CMK);
     }
 
+    public boolean isDiagnosticsServiceUnderMaintenance(String accountId) {
+        return isEntitlementRegistered(accountId, CDP_DIAGNOSTICS_SERVICE_MAINTENANCE);
```

Comment:
I think making this a positive flag fits better under entitlement structure. Straightforward: `Entitlement.CDP_VM_DIAGNOSTICS`

If it's not granted, than it's under maintenance, or there is some other issue, etc.

Reply:
ok that make sense, also would not need UMS deploy to disable the API